ts/demo-functions/src/annotate_config.ts

```diff
@@ -0,0 +1,35 @@
+import { Configs, isKubernetesObject, addAnnotation } from 'kpt-functions';
+
+const ANNOTATION_NAME = 'annotation_name';
+const ANNOTATION_VALUE = 'annotation_value';
+
+export async function annotateConfig(configs: Configs) {
+  const annotationName = configs.getFunctionConfigValueOrThrow(ANNOTATION_NAME);
+  const annotationValue = configs.getFunctionConfigValueOrThrow(
+    ANNOTATION_VALUE
+  );
+  configs
+    .get(isKubernetesObject)
```

Comment:
why do we need this? Shouldn't all objects already be valid KubernetsObject?

Reply:
We need this to make configs into an array. Otherwise I get error TS 2339: "Property 'forEach' does not exist on type 'Configs'."

Reply:
getAll()?